### PR TITLE
Move configure step from meta-xt-common

### PR DESCRIPTION
**Pull request by dsemenets**
Use default config options from xt_defconfig

### CODE REVIEW

**Review by lorc — COMMENTED** [review]
Please remove that empty line and expand commit description. It is good to mention from where you got that xt_defconfig file

File: meta-xt-rcar-driver-domain/recipes-extended/xen/xen_git.bbappend [hunk 1]
```diff
@@ -8,3 +12,4 @@ do_deploy:append () {
         ln -sfr ${DEPLOYDIR}/xen-${MACHINE}.uImage ${DEPLOYDIR}/xen-uImage
     fi
 }
+
```
[the remark this answers is not in the document]
Comment (dsemenets) [reply meta-xt-rcar-driver-domain/recipes-extended/xen/xen_git.bbappend]: Removed

**Review by lorc — APPROVED** [review]
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Review by lorc — APPROVED** [review]
`Acked-by: Volodymyr Babchuk <volodymr_babchuk@epam.com>`

**Comment by lorc** [comment]
But we need to test this.

**Comment by dsemenets** [comment]
I guess I have better solution. I have removed 'Acked-by' from @lorc. Please review

**Review by lorc — APPROVED** [review]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

Could you please explicitly state in this comments  that you tested this PR?

**Comment by dsemenets** [comment]
I successfully built prod-devel for h3ulcb and spider.
Investigation of xen config shown CONFIG_TEE and CONFIG_OPTEE are not set because they depends on CONFIG_UNSUPPORTED which didn't set by xt_defconfig

**Pull request merged**